python/tk_desktop/rpc.py

```diff
@@ -219,41 +334,48 @@ def __init__(self, pipe, authkey):
             family = "AF_PIPE"
         else:
             family = "AF_UNIX"
-        logger.debug("client connecting to to %s", pipe)
+        logger.debug("multi client connecting to to %s", pipe)
         self._connection = multiprocessing.connection.Client(
             address=pipe, family=family, authkey=authkey
         )
-        logger.debug("client connected to %s", pipe)
+        logger.debug("multi client connected to %s", pipe)
 
     def call_no_response(self, name, *args, **kwargs):
-        msg = "client calling '%s(%s, %s)'" % (name, args, kwargs)
+        msg = "multi client calling '%s(%s, %s)'" % (name, args, kwargs)
         if self._closed:
-            raise EOFError("closed " + msg)
+            raise RuntimeError("closed " + msg)
```

Comment:
No-one was listening for these specific errors. It makes more sense to use RuntimeError's here as we're not overloading an existing exception type and it helps discern unexpected errors from expected ones simply by looking at the exception type.